app/components/primer/alpha/tool-tip-element.ts

```diff
@@ -253,15 +263,17 @@ class ToolTipElement extends HTMLElement {
     if (name === 'id' || name === 'data-type') {
       if (!this.id || !this.control) return
       if (this.type === 'label') {
-        this.control.setAttribute('aria-labelledby', this.id)
+        let labelledBy = this.control.getAttribute('aria-labelledby')
+        labelledBy ? (labelledBy = `${labelledBy} ${this.id}`) : (labelledBy = this.id)
```

Comment:
I'd like us to avoid ternary statements, as they can be a little unwieldy.
```suggestion
        if (labelledBy) {
          labelledBy = `${labelledBy} ${this.id}`
        } else {
          labelledBy = this.id
        }
```